frontend/packages/console-app/src/components/cloud-shell/cloud-shell-utils.ts

```diff
@@ -86,3 +87,13 @@ export const newCloudShellWorkSpace = (
     },
   },
 });
+
+export const makeTerminalInitCalls = (username, wname, wnamespace) => {
```

Comment:
```suggestion
export const initializeTerminal = (username: string, workspaceName: string, namespace: string): Promise<...> => {
```
`Promise` should be properly typed.

Reply:
added type